Tasks/WindowsMachineFileCopy/WindowsMachineFileCopyJob.ps1

```diff
@@ -1,38 +0,0 @@
-$CopyJob = {
```

Comment:
For copying in parallel the main script still seem to use CopyJob script. Now that you are removing here, where is the script coming from?

Reply:
I didn't find any usage of this script . $copy job is being used from RoboCopyJob.ps1